faculty_cli/client.py

```diff
@@ -14,19 +14,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import json
 from contextlib import contextmanager
-
+import json
```

Comment:
`import json` should be above ` from contextlib import contextmanager`, and there should be a blank line between standard library and third party import groups.